sequence-points/p2.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(void)
+{
+
+	unsigned int x = 15;
+	unsigned int y = 25;
+	unsigned result = 0;
+
+	/* programmer assumes left to right evaluation,
+	   even with unary ops mixed in */
+	/* 16 + 15 + (50 * 50) + 50 + 15 */
+	result = x = y = x++ + x-- + (y*=2 * y) + y + x ;
```

Comment:
I'm not sure about your interpretation here (I tried multiple times to come up with a good example like this and also had a hard time). Can you refactor into simpler UBs such as i = i++?